src/AuthReg.cc

```diff
@@ -30,18 +32,18 @@
 void
 Auth::Init()
 {
-    debugs(29,DBG_IMPORTANT,"Startup: Initializing Authentication Schemes ...");
+    debugs(29, Important(69), "Startup: Initializing Authentication Schemes ...");
 #if HAVE_AUTH_MODULE_BASIC
-    static const char *basic_type = Auth::Basic::Scheme::GetInstance()->type();
-    debugs(29,DBG_IMPORTANT,"Startup: Initialized Authentication Scheme '" << basic_type << "'");
+    Assure(Basic::Scheme::GetInstance());
+    debugs(29, Important(70), "Startup: Initialized Authentication Scheme 'basic'");
 #endif
 #if HAVE_AUTH_MODULE_DIGEST
-    static const char *digest_type = Auth::Digest::Scheme::GetInstance()->type();
-    debugs(29,DBG_IMPORTANT,"Startup: Initialized Authentication Scheme '" << digest_type << "'");
+    Assure(Digest::Scheme::GetInstance());
+    debugs(29, Important(71), "Startup: Initialized Authentication Scheme 'digest'");
 #endif
 #if HAVE_AUTH_MODULE_NEGOTIATE
-    static const char *negotiate_type = Auth::Negotiate::Scheme::GetInstance()->type();
-    debugs(29,DBG_IMPORTANT,"Startup: Initialized Authentication Scheme '" << negotiate_type << "'");
+    Assure(Negotiate::Scheme::GetInstance());
+    debugs(29, Important(72), "Startup: Initialized Authentication Scheme 'negotiate'");
```

Comment:
You may recall I have been working on making these modules be initialized by Runners. I was planning to downgrade the level to 2+ at that time.

It seems a bit out of scope to do that in this PR, but I wont object if you want to include. Especially since it will allow merging message ID 68 to 71.

Reply:
Glad we seem to agree on all counts! Done in 321c28d.